src/dialect/mod.rs

```diff
@@ -143,6 +143,10 @@ pub trait Dialect: Debug + Any {
     fn supports_start_transaction_modifier(&self) -> bool {
         false
     }
+    /// Returns true if the dialect supports identifiers that start with a numeric prefix
```

Comment:
Could you please add an example of such an identifier? (e.g. `123Column`)?

Reply:
Our scenario will use numbers as business ids. The rule for table names is the business id plus an underscore followed by the specific table name. 
eg：59901_user_login